src/main/resources/assets/js/components/TaskDetailComponent.jsx

```diff
@@ -80,11 +86,8 @@ define([
             <li className="active">{task.get("id")}</li>
           </ol>
           <h5>Task Details</h5>
+          {fetchError}
           {
```

Comment:
Let's remove the unnecessary brackets here and move the markup below back in line